pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1071,8 +1072,14 @@ func TestPodTimestamp(t *testing.T) {
 			queue.MoveAllToActiveQueue()
 		}
 	}
+	backoffPod := func(pInfo *podInfo) operation {
+		return func() {
+			queue.backoffPod(pInfo.pod)
+		}
+	}
 	flushBackoffQ := func() operation {
 		return func() {
+			queue.clock.(*clock.FakeClock).Step(queue.podBackoff.MaxDuration())
```

Comment:
It is better to `Step` 2 seconds here instead of `MaxDuration`. That would be a more precise test.

Reply:
Done thanks